doc/reference/storage_lvm.md

```diff
@@ -1,45 +1,62 @@
 (storage-lvm)=
 # LVM - `lvm`
 
- - Uses LVs for images, then LV snapshots for instances and instance snapshots.
- - The filesystem used for the LVs is ext4 (can be configured to use xfs instead).
- - By default, all LVM storage pools use an LVM thinpool in which logical
-   volumes for all LXD storage entities (images, instances, etc.) are created.
-   This behavior can be changed by setting "lvm.use\_thinpool" to "false". In
-   this case, LXD will use normal logical volumes for all non-instance
-   snapshot storage entities (images, instances, etc.). This means most storage
-   operations will need to fallback to rsyncing since non-thinpool logical
-   volumes do not support snapshots of snapshots. Note that this entails
-   serious performance impacts for the LVM driver causing it to be close to the
-   fallback DIR driver both in speed and storage usage. This option should only
-   be chosen if the use-case renders it necessary.
- - For environments with high instance turn over (e.g continuous integration)
-   it may be important to tweak the archival `retain_min` and `retain_days`
-   settings in `/etc/lvm/lvm.conf` to avoid slowdowns when interacting with
-   LXD.
-
-## Storage pool configuration
+{abbr}`LVM (Logical Volume Manager)` is a storage management framework rather than a file system.
+It is used to manage physical storage devices, allowing you to create a number of logical storage volumes that use and virtualize the underlying physical storage devices.
+
+Note that it is possible to over-commit the physical storage in the process, to allow flexibility for scenarios where not all available storage is in use at the same time.
+
+## Terminology
+
+LVM can combine several physical storage devices into a *volume group*.
+You can then allocate *logical volumes* of different types from this volume group.
+
+One supported volume type is a *thin pool*, which allows over-committing the resources by creating volumes that are larger than the available physical storage.
```

Comment:
"One supported volume type is a *thin pool*, which allows over-committing the resources by creating thinly provisioned volumes whose total allocated size is larger than the available physical storage."